nodebook/gitbook/basic/1.intro.md

```diff
@@ -134,9 +140,12 @@ Git引入了`暂存区`的概念，并由此约定所有需要提交到仓库的
 
 我们可以通过diff命令来验证两个命令的输出是完全相同的：
 
-    $ diff <(git log  head -1) <(git log  head -1)
+    $ diff (git log  head -1) (git log  6bc5e700ef6cd8c1b3ed3dd015ab8f4e9df67411 -1)
```

Comment:
：）我用的是git shell。



一方 
Sarah
 
发件人： 1000copy
发送时间： 2016-05-21 22:06
收件人： 1000copy/learningnode
抄送： SarahCla; Author
主题： Re: [1000copy/learningnode] 第一次尝试提交 (#1)
In nodebook/gitbook/basic/1.intro.md:
> @@ -134,9 +140,12 @@ Git引入了`暂存区`的概念，并由此约定所有需要提交到仓库的
>  
>  我们可以通过diff命令来验证两个命令的输出是完全相同的：
>  
> -    $ diff <(git log  head -1) <(git log  head -1)
> +    $ diff (git log  head -1) (git log  6bc5e700ef6cd8c1b3ed3dd015ab8f4e9df67411 -1)

不用> 也可？我这里要报错。
$diff (git log head -1) (git log head -1)
-bash: syntax error near unexpected token `git'
你的shell是啥？
—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub